### PR TITLE
Improve error message when import case file with bad extension

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**

### CODE REVIEW

File: src/main/java/org/gridsuite/explore/server/ExploreException.java [hunk 1]
```diff
@@ -18,6 +18,8 @@ public enum Type {
         UNKNOWN_ELEMENT_TYPE,
         REMOTE_ERROR,
         IMPORT_CASE_FAILED,
+
```
Comment (bendaamerahm) [inline src/main/java/org/gridsuite/explore/server/ExploreException.java:21]: unuseful line

File: src/main/java/org/gridsuite/explore/server/services/CaseService.java [hunk 2]
```diff
@@ -68,6 +68,9 @@ UUID importCase(MultipartFile multipartFile) {
         try {
             caseUuid = restTemplate.postForObject(caseServerBaseUri + "/" + CASE_SERVER_API_VERSION + "/cases/private", request, UUID.class);
         } catch (HttpStatusCodeException e) {
+            if (e.getStatusCode().equals(HttpStatus.UNPROCESSABLE_ENTITY)) {
```
Comment (bendaamerahm) [inline src/main/java/org/gridsuite/explore/server/services/CaseService.java:71]: create test  with bad extension  to test this block if (e.getStatusCode().equals(HttpStatus.UNPROCESSABLE_ENTITY)) {}

**Review by bendaamerahm — CHANGES_REQUESTED** [review]

**Review by archendel — DISMISSED** [review]

File: src/main/java/org/gridsuite/explore/server/ExploreException.java (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/explore/server/ExploreException.java]: ok

File: src/main/java/org/gridsuite/explore/server/services/CaseService.java (hunk 2 above)
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/explore/server/services/CaseService.java:71]: ok

**Review by bendaamerahm — APPROVED** [review]

**Review by archendel — APPROVED** [review]

**Pull request merged**